src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -440,6 +461,33 @@ private static void AddProcessRegistry(IServiceCollection services)
             }
         }
 
+        private static IDistributedLockManager GetBlobLockManagerOverride(IServiceProvider provider, ScriptApplicationHostOptions applicationHostOptions)
```

Comment:
You should be able to inline this method above. Otherwise; rename to something like `GetBlobLockManager`

Reply:
Renamed in preference of the method over inline block.